### PR TITLE
Add Latency Monitor to Speedtest

### PR DESCRIPTION
**Pull request by lslamp**
I have added a monitor for ISP Latency Monitoring.

### CODE REVIEW

**Comment by lslamp** [comment]
I am very sorry but I am not a developer and am very new to this process.
I tried to have all my changes added to this pull request but I am not sure if I managed to get this done.
I also don't really understand but I have created a fork and updated the necessary files there, which are not visible to in this. No clue why.

The two files to edit are 
/etc/zabbix/bin/zbx-speedtest.sh
/etc/zabbix/zabbix_agentd.d/speedtest.conf 
 Lawrence

**Comment by lslamp** [comment]
Added item Latency to the template.
Lawrence

File: zbx-speedtest.sh [hunk 1]
```diff
@@ -99,6 +104,10 @@ do
       ACTION=show_ping
       shift
       ;;
+    -l|--latency)
+      ACTION=show_timestamp
```
Comment (pschmitt) [inline zbx-speedtest.sh:108]:
```suggestion
      ACTION=show_latency
```

**Comment by pschmitt** [comment]
Thanks. You are almost there, Lawrence..!
Just 2 things are missing before this can be merged:

1. See my comment in the review above (`show_timestamp` -> `show_latency`)
2. After fixing the above you need  to run `./generate-debian-files.sh` and `git add zbx-speedtest-debian.sh` (and commit).

**Comment by pschmitt** [comment]
So uhm. This is already implemented.

**Pull request closed without merging**